src/Drupal/DrupalExtension/Context/RawDrupalContext.php

```diff
@@ -283,7 +273,6 @@ public function userCreate($user) {
    */
   public function termCreate($term) {
     $this->dispatchHooks('BeforeTermCreateScope', $term);
-    $this->parseEntityFields('taxonomy_term', $term);
```

Comment:
Oops sorry, that was not my intention. I though I checked the commit first. Ill revert asap.